### PR TITLE
[MM-29959] restrict 'Manage Roles' button to system admins

### PR DESCRIPTION
**Pull request by maxerenberg**
#### Summary
This PR only shows the 'Manage Roles' button under the Users section of the System Console if the current user is a system admin.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-29959

### CODE REVIEW

**Review by sbishel — APPROVED** [review]
One minor comment, otherwise LGTM

File: components/admin_console/system_users/system_users_dropdown/system_users_dropdown.tsx [hunk 1]
```diff
@@ -491,6 +491,7 @@ export default class SystemUsersDropdown extends React.PureComponent<Props, Stat
         let showManageTeams = true;
         let showRevokeSessions = true;
         const showMfaReset = this.props.mfaEnabled && Boolean(user.mfa_active);
+        const showManageRoles = !isGuest && Utils.isSystemAdmin(currentUser.roles);
```
[the remark this answers is not in the document]
Comment (maxerenberg) [reply components/admin_console/system_users/system_users_dropdown/system_users_dropdown.tsx:494]: Good point.

**Review by BenCookie95 — APPROVED** [review]
LGTM, nice job!

**Review by BenCookie95 — APPROVED** [review]

**Review by sbishel — APPROVED** [review]

**Comment by furqanmlk** [comment]
image
image
image
image

All permissions are passed as expected.

**Review by furqanmlk — APPROVED** [review]

**Pull request merged**